backend/src/routes.ts

```diff
@@ -100,4 +101,6 @@ server.get('/api/v1/admin/logs', ...getApplicationLogs)
 server.get('/api/v1/admin/logs/build/:buildId', ...getItemLogs)
 server.get('/api/v1/admin/logs/approval/:approvalId', ...getItemLogs)
 
+server.get('/api/v1/export/:uuid/:deploymentId/version/:version', ...exportModel)
```

Comment:
We can get the uuid from the deploymentId, so we don't actually need the uuid value here.